### PR TITLE
Linear projection: Improve GUI

### PR DESCRIPTION
**Pull request by janezd**
*You may want to check the two commits separately. The second is just cosmetics.*

Linear projection had a weird check box.

screen shot 2016-05-22 at 22 06 20

Also, this check box is enabled/disabled when selecting a discrete/continuous variable in the color combo a few lines higher up.

I fixed the checkbox and moved it below the color combo. This made the opacity slider look weird. Besides, the opacity belongs to the color setting. I hence ended up with

screen shot 2016-05-22 at 22 11 38

I also made it a bit narrower by decreasing the widths of the combos.

### CODE REVIEW

File: Orange/widgets/visualize/owlinearprojection.py [hunk 1]
```diff
@@ -104,16 +103,14 @@ def paint(self, painter, option, widget=None):
 
 
 class AxisItem(pg.GraphicsObject):
-    def __init__(self, parent=None, line=None, label=None, **kwargs):
-        super().__init__(parent, **kwargs)
+    def __init__(self, parent=None, line=None, label=None, *args):
```
Comment (astaric) [inline Orange/widgets/visualize/owlinearprojection.py]: Is replacing **kwargs with *args really just a lint change?

File: Orange/widgets/visualize/owlinearprojection.py [hunk 2]
```diff
@@ -422,7 +424,7 @@ def icon(name):
                 "Zoom to fit", self, icon=icon("zoom_reset"),
                 shortcut=QKeySequence(Qt.ControlModifier | Qt.Key_0),
                 triggered=lambda:
-                    self.viewbox.setRange(QRectF(-1.05, -1.05, 2.1, 2.1))),
+                self.viewbox.setRange(QRectF(-1.05, -1.05, 2.1, 2.1))),
```
Comment (astaric) [inline Orange/widgets/visualize/owlinearprojection.py]: This indent does not seem right

**Comment by lanzagar** [comment]
Since you are working on Linear Projection, could you make sure that Report works?
It does not for me.

**Comment by lanzagar** [comment]
Merging this, and report can be handled in another PR.

**Comment by janezd** [comment]
@lanzagar, I checked this: `PngFormat._save_buffer` (which contains only a single line, `buffer.save(filename, "png")`), freezes.

**Pull request merged**